src/query/interpreter.cpp

```diff
@@ -2910,18 +2910,21 @@ PreparedQuery PrepareInfoQuery(ParsedQuery parsed_query, bool in_explicit_transa
       };
       break;
     case InfoQuery::InfoType::INDEX:
-      header = {"index type", "label", "property"};
+      header = {"index type", "label", "property", "count"};
       handler = [interpreter_context] {
         auto *db = interpreter_context->db.get();
         auto info = db->ListAllIndices();
         std::vector<std::vector<TypedValue>> results;
         results.reserve(info.label.size() + info.label_property.size());
+        auto acc = db->Access();
         for (const auto &item : info.label) {
-          results.push_back({TypedValue("label"), TypedValue(db->LabelToName(item)), TypedValue()});
+          results.push_back({TypedValue("label"), TypedValue(db->LabelToName(item)), TypedValue(),
+                             TypedValue(static_cast<int>(acc->ApproximateVertexCount(item)))});
```

Comment:
How correct is this number, by how much it can miss?

Reply:
It depends, it's the number of non-deleted elements inside the skiplist, for the sake of analysis from the users perspective should be accurate enough -> resolving this for now 😃